INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
@simison 

another check happens later when reaching
`modules/core/server/services/...../influxService.getClient()`. I'll wrap the error login inside: `if (process.env.NODE_ENV === 'production')`. how to do this globally for the future influxdb modules i don't know.

the purpose of the early check is to prevent unnecessary work of collecting the data when influxdb is disabled.

I'll move the check to the beginning of the message-to-influx service unless you ask for something else.